meh/ui/text.py

```diff
@@ -21,9 +21,16 @@
 
 from meh import MAIN_RESPONSE_DEBUG, MAIN_RESPONSE_SAVE, MAIN_RESPONSE_SHELL, MAIN_RESPONSE_QUIT
 from meh.ui import AbstractIntf, AbstractSaveExceptionWindow, AbstractMainExceptionWindow, AbstractMessageWindow
-import report
-import report.io.TextIO
-from report import LIBREPORT_WAIT, LIBREPORT_RUN_CLI
+
+
+LIBREPORT_AVAILABLE = False
+try:
+    import report
+    import report.io.TextIO
+    from report import LIBREPORT_WAIT, LIBREPORT_RUN_CLI
+    LIBREPORT_AVAILABLE = True
+except ImportError:
+    print("python-meh GUI: libreport is not available in this environment")
```

Comment:
did you mean "python-meh TUI: ..." ?

Reply:
Good point - lets fix that & actually tell the user that this will result in bug reporting being disabled. :)